test/csv/test_data_converters.rb

```diff
@@ -103,4 +103,19 @@ def test_builtin_date_time_converter_iso8601_utc
     assert_equal(datetime,
                  CSV::Converters[:date_time][iso8601_string])
   end
+
+  def test_builtin_date_time_converter_near_iso
```

Comment:
Could you split this test to multiple tests like existing `test_bulitin_date_time_converter_iso8601_*` for easy to debug on failure?
If `"2018-01-14 22:25"` is failed, the rest cases such as `"2018-01-14 22:25:19"` aren't executed by this style. The results of all cases are helpful to debug.